demo/public/app.js

```diff
@@ -2,6 +2,7 @@ require('file-loader?name=index.html!./index.html');
 require('./app.css');
 
 const fhirpath = require('../../build/fhirpath.js');
+const fhirpath_r4 = require('../../fhir-context/r4');
```

Comment:
See fhirpath_r4 in the README.

Reply:
functionally there is nothing wrong, but it would help to have the word "model" in the name, especially for global variables. Otherwise I would have assumed fhirpath_r4 is fhirpath for R4, and fhirpath is probably for some default version of fhirpath.

Reply:
Changed.